### PR TITLE
Sane limits for certain EVM parameters

### PR DESCRIPTION
**Pull request by axic**

### CODE REVIEW

**Comment by axic** [comment]
Turns out a similar proposal was made in #106. While this proposal doesn't explicitly limit the block gas limit, it does limit transaction gas limits.

File: EIPS/eip-1985.md [hunk 1]
```diff
@@ -74,6 +74,15 @@ Using the `2**63 - 1` constant to limit some of the ranges:
...
 - makes arithmetic simpler (e.g. checking out-of-gas conditions is simple as `gas_counter < 0`).
 
+### Timestamp
+
+The [Yellow Paper] defines the timestamp in block as "A scalar value equal to the reasonable output of Unix’s time() at this block’s inception".
+IEEE Std 1003.1-2001 (POSIX.1) leaves that definition implementation defined.
+
+### Comparing current implementations
+
+- Timestamp is implemented as a 64-bit value in [geth] and [Parity]
```
Comment (chfast) [inline EIPS/eip-1985.md]: Also in Aleth, at least around EVM: https://github.com/ethereum/aleth/pull/4797

File: EIPS/eip-1985.md [hunk 2]
```diff
@@ -0,0 +1,118 @@
...
+
+TBA
+
+## References
+
+[EIP-106](https://github.com/ethereum/EIPs/issues/106) proposed the block gas limit to be limited at `2**63 - 1`.
+
+## TODO
+
+1. The size of addresses are specified in Yellow Paper,
```
Comment (chfast) [inline EIPS/eip-1985.md]: This can be moved to Rationale section.

File: EIPS/eip-1985.md [hunk 3]
```diff
@@ -0,0 +1,118 @@
...
+
+## References
+
+[EIP-106](https://github.com/ethereum/EIPs/issues/106) proposed the block gas limit to be limited at `2**63 - 1`.
+
+## TODO
+
+1. The size of addresses are specified in Yellow Paper, 
+e.g. COINBASE is specified as returning H_c which has 20 bytes.
+2. Do the gas limit applies for the gas param in CALL?
```
Comment (chfast) [inline EIPS/eip-1985.md]: Remove it? What solidity currently provides for calls?

Comment (axic) [reply EIPS/eip-1985.md]: I'd say it applies. Then it can be discussed on the ACD calls.

**Pull request merged**